aioshelly/rpc_device/device.py

```diff
@@ -338,6 +339,8 @@ async def get_all_pages(self, first_page: dict[str, Any]) -> dict[str, Any]:
 
     async def script_list(self) -> list[ShellyScript]:
         """Get a list of scripts from 'Script.List'."""
+        if self.model == MODEL_X_BASE:
+            return []
```

Comment:
We don't call `script_list` anywhere during init in `aioshelly` we call it from Shelly integration when we setup Bluetooth, we should completely disable the usage of Bluetooth same as we do for sleeping devices and for the wall display:
https://github.com/home-assistant/core/blob/1a394876b11c85893998595425244e84f157abe8/homeassistant/components/shelly/config_flow.py#L458-L464